mnemosyne/web_server/review_wdgt.py

```diff
@@ -186,7 +186,7 @@ def to_html(self):
             self.answer = "&nbsp;"
 
         hide_answer_css = ""
-        if card.fact_view.a_on_top_of_q:
+        if None != card and card.fact_view.a_on_top_of_q:
```

Comment:
The idiomatic way to check if something is not None in Python is `card is not None`

Reply:
Okay, thanks.